README.md

```diff
@@ -121,6 +121,18 @@ Note that there is a currently a race condition that means that the client test
 some server test suite requests. As a work-around, run the server test suite a second time while the
 client is waiting for clinical data requests during test 1.2.
 
+## Running the Bulk Export Server Tests
+
+These tests require read-write access to FHIR server with `Patient/$export` enabled, or with permissions to those actions assigned to a Bulk Data Authorization Access Token. To view an example run of these tests:
```

Comment:
Why do they require write access? Is that a HAPI requirement around bulk export? I think all of this is in the context of our reference server, so I would make that clear:
```suggestion
The Server Bulk Export tests (group 1.4) can be run against the Inferno Reference Server as long as it is running with write enabled:
```

Reply:
You're right this is only for our reference server not from the FHIR spec